<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-03-28T21:20:40 IST" name="Suite" finished-at="2022-03-28T21:21:08 IST" duration-ms="28726">
    <groups>
    </groups>
    <test started-at="2022-03-28T21:20:40 IST" name="Test" finished-at="2022-03-28T21:21:08 IST" duration-ms="28726">
      <class name="report.TestRun">
        <test-method is-config="true" signature="configBS()[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:20:39 IST" name="configBS" finished-at="2022-03-28T21:20:40 IST" duration-ms="619" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBS -->
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:20:40 IST" name="launchBrowser" finished-at="2022-03-28T21:20:45 IST" duration-ms="4763" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method is-config="true" signature="login()[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:20:45 IST" name="login" finished-at="2022-03-28T21:20:51 IST" duration-ms="5980" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="test1()[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:20:51 IST" name="test1" finished-at="2022-03-28T21:20:58 IST" duration-ms="6910" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method is-config="true" signature="logOut(org.testng.ITestResult)[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:20:58 IST" name="logOut" finished-at="2022-03-28T21:20:59 IST" duration-ms="1747" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=test1 status=SUCCESS method=TestRun.test1()[pri:0, instance:report.TestRun@60b4beb4] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method is-config="true" signature="login()[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:20:59 IST" name="login" finished-at="2022-03-28T21:21:01 IST" duration-ms="1784" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="test2()[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:21:01 IST" name="test2" finished-at="2022-03-28T21:21:01 IST" duration-ms="8" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at report.TestRun.test2(TestRun.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method is-config="true" signature="logOut(org.testng.ITestResult)[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:21:01 IST" name="logOut" finished-at="2022-03-28T21:21:04 IST" duration-ms="2561" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=test2 status=FAILURE method=TestRun.test2()[pri:0, instance:report.TestRun@60b4beb4] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method is-config="true" signature="login()[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:21:04 IST" name="login" finished-at="2022-03-28T21:21:05 IST" duration-ms="1723" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="test3()[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:21:05 IST" name="test3" finished-at="2022-03-28T21:21:05 IST" duration-ms="4" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test3 got skipped]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test3 got skipped
at report.TestRun.test3(TestRun.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method is-config="true" signature="logOut(org.testng.ITestResult)[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:21:05 IST" name="logOut" finished-at="2022-03-28T21:21:08 IST" duration-ms="2848" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=test3 status=SKIP method=TestRun.test3()[pri:0, instance:report.TestRun@60b4beb4] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method is-config="true" signature="quitBrowser()[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:21:08 IST" name="quitBrowser" finished-at="2022-03-28T21:21:08 IST" duration-ms="192" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitBrowser -->
        <test-method is-config="true" signature="configAS()[pri:0, instance:report.TestRun@60b4beb4]" started-at="2022-03-28T21:21:09 IST" name="configAS" finished-at="2022-03-28T21:21:09 IST" duration-ms="656" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configAS -->
      </class> <!-- report.TestRun -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
